dwave/cloud/__init__.py

```diff
@@ -128,12 +137,18 @@ class Formatter(FilteredSecretsFormatter, formatter_base):
     else:
         Formatter = formatter_base
 
+    if not additive:
+        # make sure handlers are not accumulated
+        while len(logger.handlers):
```

Comment:
This makes me slightly nervous in case there are some edge cases where `removeHandler()` doesn't actually remove anything. But probably safe.

Reply:
That function sounds more sophisticated than [it really is](https://github.com/python/cpython/blob/0315fdc24df9be03ae6ba40bac25b512e1e8da0a/Lib/logging/__init__.py#L1713). :slightly_smiling_face: